adminapi/request.py

```diff
@@ -91,7 +91,14 @@ def calc_signature(private_key, timestamp, data=None):
     Returns the signature as base64 encoded unicode, ready for transport.
     """
     message = calc_message(timestamp, data)
-    sig = private_key.sign_ssh_data(message.encode())
+    try:
+        sig = private_key.sign_ssh_data(message.encode())
+    except SSHException as e:
+        # There is a possbility, that the key cannot be used for authentication
+        # e.g., RSA keys for PIV Attestation
+        if e.args[0] == 'key cannot be used for signing':
+            return ""
```

Comment:
I am not sure if returning an empty string here is a good idea. I am not even sure if this can be processed by the backend or if it would reject it. Can you please check ?